# This srv will return a section msg for every section in the road
---
Section[] sections
